hal/include/FreeRTOS.h

```diff
@@ -1,1063 +1,1063 @@
-/*
```

Comment:
Did you save your HALCoGen project after generating the code? It should modify the `onboarding.dil` and `onboarding.hcg` files.

Reply:
I saved the project after generating the code but I'm not sure. Should I redo the code-generating process so that it can modify the files?

Reply:
That's weird lol. Yeah maybe try it again.